tensorflow/g3doc/how_tos/adding_an_op/index.md

```diff
@@ -107,7 +107,7 @@ REGISTER_KERNEL_BUILDER(Name("ZeroOut").Device(DEVICE_CPU), ZeroOutOp);
 ```
 
 Once you
-[build and reinstall TensorFlow](../../get_started/os_setup.md#pip-installation), the
+[build and reinstall TensorFlow](../../get_started/os_setup.md#create-the-pip-package-and-install), the
```

Comment:
I think create-the-pip-package-and-install is the right tag. Inside the
adding-an-op tutorial they'd already have a source install.

On Mon, Jan 4, 2016 at 9:48 PM Sam Abrahams <notifications@github.com>
wrote:

> In tensorflow/g3doc/how_tos/adding_an_op/index.md
> <https://github.com/tensorflow/tensorflow/pull/683#discussion_r48814272>:
>
> > @@ -107,7 +107,7 @@ REGISTER_KERNEL_BUILDER(Name("ZeroOut").Device(DEVICE_CPU), ZeroOutOp);
> >  ```
> >
> >  Once you
> > -[build and reinstall TensorFlow](../../get_started/os_setup.md#pip-installation), the
> > +[build and reinstall TensorFlow](../../get_started/os_setup.md#create-the-pip-package-and-install), the
>
> Before making any of the {#anchor} changes on #660
> <https://github.com/tensorflow/tensorflow/pull/660>, this link originally
> pointed to this part of the file
> <https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#pip-installation>
> (which #660 <https://github.com/tensorflow/tensorflow/pull/660>
> maintained, using the GitHub style anchor tag), as opposed to this change
> which points to this part of the file
> <https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#create-th%0D%20e-pip-pa%0D%20ckage-and-install>
> several sections down. However, it looks like your change is probably going
> to a more relevant part of the page, as they will need to install from
> sources as opposed to 'easy-installing' from pip.
>
> I'm thinking maybe this line should read
>
> [build and reinstall TensorFlow](../../get_started/os_setup.md#installing-from-sources), the
>
> to point to the very start of the "Installing from Sources"
> <https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#installing-from-sources>
> section as opposed to the very end of it.
>
> Thoughts? Probably over thinking a one-line change to a doc file, but this
> piqued my curiosity.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/683/files#r48814272>.
>